### PR TITLE
updated operator to work with multiple namespaces

### PR DESCRIPTION
**Pull request by rltvty**
split the startup into pulling 2 namespaces... `operatorNs` for the namespace the operator is running in, and `watchNs` for the namespaces passed via the `WATCH_NAMESPACE` environment variable.

I've successfully tested this in my development EKS cluster.  I'm able to get the operator to deploy Cassandra datacenters in namespaces outside the operator.

### CODE REVIEW

File: operator/cmd/manager/main.go [hunk 1]
```diff
@@ -101,12 +101,18 @@ func main() {
...
 		log.Error(err, "Failed to get watch namespace")
 		os.Exit(1)
 	}
 
+	operatorNs, err := k8sutil.GetOperatorNamespace()
+	if err != nil {
+		log.Error(err, "Failed to get operator namespace")
+		os.Exit(1)
+	}
+
```
Comment (rltvty) [inline operator/cmd/manager/main.go:115]: this should never fail, but check anyway

**Comment by rltvty** [comment]
related to #115

File: operator/cmd/manager/main.go [hunk 2]
```diff
@@ -151,7 +157,7 @@ func main() {
 	// Note that this is not intended to be used for excluding namespaces, this is better done via a Predicate
 	// Also note that you may face performance issues when using this with a high number of namespaces.
 	// More Info: https://godoc.org/github.com/kubernetes-sigs/controller-runtime/pkg/cache#MultiNamespacedCacheBuilder
-	if strings.Contains(namespace, ",") {
+	if strings.Contains(watchNs, ",") {
 		options.Namespace = ""
 		options.NewCache = cache.MultiNamespacedCacheBuilder(strings.Split(namespace, ","))
```
Comment (jimdickinson) [inline operator/cmd/manager/main.go:162]: this usage of namespace wasn't fixed

**Comment by jimdickinson** [comment]
I'll also branch off of your branch and add an integration test, if that's ok :)

**Comment by rltvty** [comment]
@jimdickinson please feel free to branch off my work.  Sorry, it has taken so long for me to get back to you.

**Comment by jsanda** [comment]
Is this PR still needed since #173 was recently merged?

**Comment by jimdickinson** [comment]
@jsanda probably no

@rltvty sorry I had been intending on merging this after pinging you about signing our new CLA - the website is being adjusted right now to have less specific language about Cassandra drivers - I'm really sorry we were a little slow to get the contributor process well setup, and this sat around stagnant for some time :(

**Comment by rltvty** [comment]
@jimdickinson no worries on the delay.

**Pull request closed without merging**